Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/eventregistration/EventRegistrationUIModule.kt

```diff
@@ -25,4 +27,7 @@ internal abstract class EventRegistrationUIModule {
 
     @ContributesAndroidInjector(modules = [TraceLocationCategoryModule::class])
     abstract fun traceLocationCategoryFragment(): TraceLocationCategoryFragment
+
+    @ContributesAndroidInjector(modules = [TraceLocationCreateModule::class])
+    abstract fun traceLocationCreateFragment(): TraceLocationCreateFragment
```

Comment:
Maybe better naming such as `CreateTraceLocationFragmentModule` and `CreateTraceLocationFragment` ?

Reply:
In other features like the ContactDiary, we also have the Feature Name first for every component. And I personally like it that way, I am often searching for components (Serch Classes in Android Studio), and then it's nice to just type the feature and then you see all of the components.

Reply:
All is fine  ,but I would like still to rename the module to have the fragment name so it would be 
`TraceLocationCreateFragmentModule` So I would know it a module for this fragment

Reply:
modules are renamed